Code/GraphMol/MolStandardize/Charge.h

```diff
@@ -101,12 +101,16 @@ class RDKIT_MOLSTANDARDIZE_EXPORT Reionizer {
 class RDKIT_MOLSTANDARDIZE_EXPORT Uncharger {
  public:
   Uncharger();
+  Uncharger(bool canonicalOrdering) : Uncharger() {
```

Comment:
slightly better style:

```
Uncharger(bool canonicalOrdering) : Uncharger(), df_canonicalOrdering(canonicalOrdering)
```

Reply:
I've never seen one constructor of a class calling another constructor of the same class...

Reply:
Learn something new:
https://en.wikipedia.org/wiki/C++11#Object_construction_improvement

In this case:
```
Uncharger(bool canonicalOrdering) : Uncharger(), df_canonicalOrdering(canonicalOrdering)
```

Reply:
I tried that initially, but it caused problems with at least one compiler, so I switched to this